test/fixtures/es-modules/es-note-promiserej-import-2.cjs

```diff
@@ -0,0 +1 @@
+import('valid');
```

Comment:
nit: can we add these missing newlines for better diffs down the line?

Reply:
Added newlines in tests.